plugins/services/src/js/containers/tasks/TaskTable.js

```diff
@@ -327,19 +327,21 @@ class TaskTable extends React.Component {
     const unhealthy = task.health === TaskHealthStates.UNHEALTHY;
     const unknown = task.health === TaskHealthStates.UNKNOWN;
 
+    const failing = ["TASK_ERROR", "TASK_FAILED", "TASK_KILLING"].includes(
```

Comment:
`TASK_KILLING` doesn't mean it is failing, does it? Let's create a separate shortcut for `killing`